ProQuest.js

```diff
@@ -449,13 +471,15 @@ function scrape(doc, url, type, pdfUrl) {
 			mimeType: 'application/pdf'
 		});
 	} else {
-		var pdfLink = ZU.xpath(doc, '(//div[@id="side_panel"]//\
+		var pdfLink = ZU.xpath(doc, '//li[@id="tab-SPDF-null" or @id="tab-FulltextPreview-null"]/a');
```

Comment:
I'm also seeing "tab-ScannedPDF-null" and "tab-PageImage-null" (for Page Preview tab). But...

(1) it seems that there is always the "Download PDF" link on the side, so I don't think we need to worry about this method of fetching the PDF anymore (and we also don't need to worry about getting the PDF url from the embed node in `doWeb`

(2) if, for whatever reason, you want to continue doing this, we need to make sure that we always grab the full text PDF and not the preview. It seems like the preview tab is always after the full text tab, so this should be ok.